docker-tests.sh

```diff
@@ -111,6 +111,9 @@ start_container() {
     "${image_tag}" \
     "${init}" \
     > "${container_id}"
+  # Create an additional network to perform tests on
+  docker network create net2 --driver bridge || true
+  docker network connect net2 $(cat ${container_id})
```

Comment:
ca va foiré là si ca `|| true` avant

Reply:
If it fails when the network already exists it will still connect. (useful for running multiple testcases on the same worker)
If the network fails to create for any other reason it should fail, as it will be an issue with docker itself.